src/software/SfM/main_ComputeCCTagStructureFromKnownPoses.cpp

```diff
@@ -58,7 +377,10 @@ int main(int argc, char **argv)
     << "[-f|--match_file] (opt.) path to a matches file (used pairs will be used)\n"
```

Comment:
apparently this option has been dropped at some point but the help has not been updated